src/main.cpp

```diff
@@ -102,6 +122,30 @@ void setup()
         }
         delay(100);
     }
+    //set firingMode
+    if (pins.select1) {
+        select1.update();
+    }
+    if (pins.select2) {
+        select2.update();
+    }
+    if(select1.isPressed()){
+        firingMode = 0;
+    }
+    else if(select2.isPressed()){
+        firingMode = 2;
+    }
+    else {
+    firingMode = 1;
+    }
+    //changeFPS on boot by firing mode
+    for (int i = 0; i<numMotors; i++){
+        revRPM[i] = revRPMset[firingMode][i/2];
+        idleRPM[i] = idleRPMset[firingMode][i/2];
+        firingRPM[i] = revRPM[i]*0.9;
```

Comment:
I don't like hardcoding this firing rpm ratio so far down in the file

Reply:
Yeah me too but I'm going to redo this when I implement closed loop anyway